polaris/polaris/utils.py

```diff
@@ -135,24 +162,20 @@ def create_stellar_deposit(
         transaction.save()
         raise ValueError(transaction.status_message)
 
-    # if we don't know if the destination account exists
-    if not destination_exists:
-        try:
-            _, created, pending_trust = get_or_create_transaction_destination_account(
-                transaction
-            )
-        except RuntimeError as e:
-            transaction.status = Transaction.STATUS.error
-            transaction.status_message = str(e)
+    try:
+        _, created, pending_trust = get_or_create_transaction_destination_account(
+            transaction
+        )
+    except RuntimeError as e:
+        transaction.status = Transaction.STATUS.error
+        transaction.status_message = str(e)
+        transaction.save()
+        logger.error(transaction.status_message)
+        return False
+    if created or pending_trust:
+        if pending_trust and transaction.status != Transaction.STATUS.pending_trust:
+            transaction.status = Transaction.STATUS.pending_trust
```

Comment:
So it looks like the change here is we're no longer doing `return False`. But if the account doesn't have a trust-line and `transaction.claimable_balance_supported` is `False`, we still want to `return False`.